src/parser/jobs/drg/modules/BloodOfTheDragon.js

```diff
@@ -138,7 +181,11 @@ export default class BloodOfTheDragon extends Module {
 
 		if (!this._lifeWindows.current.nastronds.some(nastrond => nastrond.timestamp === event.timestamp)) {
 			// Dedupe Nastrond casts, since that can occasionally happen
-			this._lifeWindows.current.nastronds.push(event)
+			this._lifeWindows.current.nastronds.push({
+				...event,
```

Comment:
Certainly not necessary but since it looks like the only piece of data from the actual `event` blob you're using is the timestamp, it may actually be worth it to change `...event,` to `timestamp: event.timestamp,` and trim out all the excess so it's cleaner/more maintainable down the road. Same thing applies to the Stardiver bit ~20 lines down.